docs/source/contents/4_draft_mapping.rst

```diff
@@ -118,21 +118,143 @@ We have decided to not migrate these values as is justified above. Here's an `ex
+case of photography, the dimensions can be used to help determine the type of film.
+
+Justification
+^^^^^^^^^^^^^
+The working group's shared philosophies were influential in decided on the best property to use for <extent> values. The
+Islandora Metadata Interest Group's default mapping suggests using dcterms:extent and using a blank node with a literal as
+a RDF value. This group is against using blank nodes when at all possible because they make it more difficult for the
+user to consume content. The Samvera mapping uses rdau:extent, which is less than ideal because rdau does not support
+content negotiation. This means that the URI provided for the desired property does not allow a user to directly request
+RDF. No other more suitable properties could be found for <extent> values. Given this predicament, the working group
+decided to use rdau:extent because it is dereferenceable, which a blank node is not. Still, the inability to retrieve
+RDF directly will limit users wishing to interact with our data in this way.
+
+Xpath
+^^^^^
+
+mods:physicalDescription/mods:extent
+
+Decision
+^^^^^^^^
+`Example record - knoxgardens:125 <https://digital.lib.utk.edu/collections/islandora/object/knoxgardens%3A125/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <extent>3 1/4 x 5 inches</extent>
+
+.. code-block:: turtle
+
+    @prefix rdau: <http://rdaregistry.info/Elements/u/#> .
+
+    <https://example.org/objects/1>
+        rdau:extent.en "3 1/4 x 5 inches" .
+rdau:extent
+
+extent - @unit
+--------------
+
+Use Case
+^^^^^^^^
+The Great Smoky Mountains Colloquy collection is the only collection that includes the unit attribute on <extent>. The
+collection consists of 34 total records.
 
 Justification
 ^^^^^^^^^^^^^
+It is important for the user to know what the unit of measurement is for a value within the <extent> field. It is also
+important for us to share this information consistently. In order to retain the needed information while also conforming
+the metadata from this collection with the rest of our records, we propose that the @unit value is added to the extent
+string during migration. This would involve simply taking the existing value in <extent> and then adding ' pages' to the
+string. Note that all of the resources within the Colloquy collection have more than one page, so the plural form of the
+word will always be accurate. See the Decision section of extent above for more explanation of rdau:extent.en.
 
 Xpath
 ^^^^^
 
+mods:physicalDescription/mods:extent[@unit="pages"]
+
 Decision
 ^^^^^^^^
+`Example record - colloquy:202 <https://digital.lib.utk.edu/collections/islandora/object/colloquy%3A202/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <extent unit="pages">4</extent>
+
+.. code-block:: turtle
+
+    @prefix rdau: <http://rdaregistry.info/Elements/u/#> .
+
+    <https://example.org/objects/1>
+        rdau:extent.en "4 pages" .
```

Comment:
See the message in my review. Also, the turtle lexer does not like namespaced URIs like this. If we don't change properties, I think we should just use the full URI.